data/darktable.css.in

```diff
@@ -84,15 +84,15 @@ GtkAlignment
   background-color: shade(@selected_bg_color, 1.2);
   background-image: none;
 }
-#lib-plugin-ui .button:${GTK_CSS_CLASS},
-.button:${GTK_CSS_CLASS}
```

Comment:
I know, to accommodate different Gtk+ version. I didn't push the right version. Will fix that in a moment.